spread.yaml

```diff
@@ -8,15 +8,28 @@ environment:
 backends:
   lxd:
     systems: [ubuntu-16.04]
+  linode:
+    key: "$(HOST: echo $SPREAD_LINODE_KEY)"
+    systems:
+      - ubuntu-16.04-64:
+          workers: 3
 
 suites:
  spread_tests/:
    summary: tests of the snapcraft snap
    prepare: |
      apt update
```

Comment:
We have been suffering lots of problems regarding apt and connection issues from linode hosts to ubuntu mirrors over ipv6 (which is enabled by default), if you hit those, in the form of tests timing out, you could either disable ipv6 or give it a low priority (the latter is giving very good results to us so far https://github.com/snapcore/snapd/blob/master/spread.yaml#L294)